yt/frontends/arepo/fields.py

```diff
@@ -59,10 +59,10 @@ def setup_gas_particle_fields(self, ptype):
         # Since the AREPO gas "particles" are Voronoi cells, we can
         # define a volume here
         def _volume(field, data):
-            return data[ptype, "mass"] / data[ptype, "density"]
+            return data["gas", "mass"] / data["gas", "density"]
 
         self.add_field(
-            (ptype, "cell_volume"),
+            ("gas", "cell_volume"),
             function=_volume,
             sampling_type="local",
             units=self.ds.unit_system["volume"],
```

Comment:
Should this be changed elsewhere too?  It looks reasonable to me, since in Arepo the volume (at least right now) will only make sense for the gas particles.  Do we need to concern ourselves with potential two-fluid sims?

Reply:
By "this" you mean?

Reply:
`ptype` to `"gas"`

Reply:
I fixed this also now for the `"pressure"` field